doc/library/tensor/nnet/nnet.txt

```diff
@@ -120,12 +120,12 @@
 
    The softmax function will, when applied to a matrix, compute the softmax values row-wise.
 
-    :note: this insert a particular op. But this op don't yet
-       implement the Rop for hessian free. If you want that, implement
-       this equivalent code that have the Rop implemented
-       ``exp(x)/exp(x).sum(1, keepdims=True)``. Theano should
+    :note: this uses a particular operation. But this method doesn't yet
```

Comment:
In fact, This is outdated. softmax() now support Rop. So can you remove the part about this and keep the numerical stable part? I'll kill travis on this commit.